radis/api/kuruczapi.py

```diff
@@ -98,7 +98,7 @@ def load_pf_Barklem2016(self):
         `Barklem & Collet (2016), Table 8 <https://doi.org/10.1051/0004-6361/201526961>`_
 
         """
-        file_path="radis/levels/pfTKurucz_values.txt"
+        file_path=r"radis/levels/pfTKurucz_values.txt"
```

Comment:
Make it an absolute path. There are some functions in Radis/dB and Radis/test/utils to get proper maths independently of 1) the working directory 2) your OS